types/datadog-tracer/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
What was the problem when this was enabled?

Reply:
I get linting error in the line `https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23969/files#diff-ba3f2b1c5553bd8cdba0777ef902547fR19` and also on the import statements. Looks like this rule prevents me from declaring anything without exporting. I can't avoid this because this is definition for `CommonJS` module and have to use the `export = Tracer;` construct.

Reply:
Works on my machine without lint errors, so I'll just merge. You probably have an older install of dtslint.